.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Dear friend, we, the rest of the NativeScript community really appreciate your feedback! While we are doing all we can to take care of every issue, sometimes we get overwhelmed. Because of that, we will consider issues that are not constructive or problems that cannot be reproduced "dead". Additionally, we will treat feature requests or bug reports with unanswered questions regarding the behavior/reproduction for more than 20 days "dead". All "dead" issues will get closed.
+
+## Please, provide the details below:
+
+### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
+
+### Tell us about the problem
+Please, ensure your title is less than 63 characters long and starts with a capital
```

Comment:
This line doesn't feel like it belongs here. Perhaps move it to the top of the template in _italic_?